src/Page/Simulator.elm

```diff
@@ -431,6 +431,7 @@ lifeCycleStepsView db displayMode simulator =
             (\index current ->
                 StepView.view
                     { db = db
+                    , inputs = simulator.inputs
                     , detailed = displayMode == DetailedMode
                     , index = index
                     , product = simulator.inputs.product
```

Comment:
We don't need to expose `product` anymore as we now pass `inputs`.

Reply:
Fixed in https://github.com/MTES-MCT/wikicarbone/pull/39/commits/1b42725f342ad46651f572ff63e3da664b20fe39